discovery-provider/src/app.py

```diff
@@ -437,6 +438,10 @@ def configure_celery(flask_app, celery, test_config=None):
                 "task": "index_challenges",
                 "schedule": timedelta(seconds=5),
             },
+            "index_oracles": {
+                "task": "index_oracles",
+                "schedule": timedelta(seconds=60),
```

Comment:
maybe we should do like 5 minutes. i dont see a reason we need it so often

Reply:
I super agree